tests/test_repr.py

```
@@ -0,0 +1,174 @@
+
+
+class TestQdrantClientRepr:
+    """QdrantClient (the sync facade)."""
+
+    def test_local_in_memory(self):
+        client = QdrantClient(":memory:")
+        r = repr(client)
+        assert r.startswith("<QdrantClient mode=local location=':memory:'>")
+
+    def test_local_path(self):
+        client = QdrantClient(path="/tmp/qdrant")
```

Ruff (0.15.21)

[error] Probable insecure usage of temporary file or directory: "/tmp/qdrant"

(S108)